### DISCUSSION TITLE
Medusa fails to start, error validators.btc_address

### DISCUSSION THREAD

**Discussion by ibuk01** [pymedusa/Medusa#9072] [General]
Since release 0.5.5 the daemon does no longer start up and needs 'validators.btc_address'.
Version 0.5.4 still launches without any issues. I disabled automatic updates for now.

I remember that Sickbeard or Sickrage also became a victim of such illegal actions of such .... gangsters.

Kind regards.

**Comment by OmgImAlexis** [comment]
Nope looks like it's part of the validators package which was updated. https://github.com/pymedusa/Medusa/blob/c136c8c7bb1e44993e07ff40cad9374e98fc942d/ext/validators/btc_address.py

We're not in the business of trying to scam our users like some of the ex-members of this group.

**Reply by OmgImAlexis** [reply]
@medariox any ideas?

**Comment by medariox** [comment]
What? Please don't spread false rumors. That `validators.btc_address` is just part of a bigger library and is just used to validate a btc address. We don't use that part of the code, but it is included in the package validators which is used to validate various kinds of things (emails, domains, etc. - that's what we use it for). If you're getting that error it means your update didn't finish smoothly. I suggest you pull the new version again and see if it fixes it. Also please don't bring us in connection to Sick* devs, we have nothing to do with them and never will!

**Reply by ibuk01** [reply]
huge sorry for bringing up these dark ages :) 

So I simply cannot launch the latest branch, which sounds much better! I even tried downloading the 0.5.5 using wget (the archive URL) and GIT into a new directory, but it didn't launch on my Debian Buster. I also removed my users .local/lib/python3.7 directory and installed all the pip-packages (requirements.txt) again but it didn't help too.

**Reply by medariox** [reply]
I don't know which setup you have in place, but generally you should try get the new version again. You can do it with git for example. That will most likely fix your issue.

**Reply by ibuk01** [reply]
will try it tomorrow / thanks for reacting that quick. 
and I am realy sorry for bringing that up again!

best regards.

**Reply by bassings** [reply]
I had a similar problem. I tried a bunch of stuff to get it working, eventually fixed it with a simple: 
pip remove validators
pip install validators

**Comment by medariox** [comment]
1. Remove any conflicting validators versions:
`pip uninstall validators`
or
`pip3 uninstall validators`

2. Delete the `__pycache__` folder in `Medusa\ext\validators`.

If the above doesn't work your update went wrong:
3. Get a new copy of the latest version via `git` or manually.

**Reply by ibuk01** [reply]
works now - thanks alot.
